src/Components/PreviewBlocks/PreviewBlocks.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import clinic_img from "./../../assets/Asset 26.svg";
+import gavel_img from "./../../assets/Asset 31.svg";
+import speaker_img from "./../../assets/Asset 28.svg";
+
+import "./PreviewBlocks.scss";
+
+const PreviewBlocks = () => {
+  return (
+    <>
+      <div className="preview">
+        <img src={clinic_img} alt="image-of-uterus" className="icon"></img>
```

Comment:
alt tag could be either "uterus" or "icon of uterus", no need for hyphens since alt tags are for screen readers. 

(using an image in the alt tag is breaking the CI build, so just wanted to call out alternative language)

Reply:
Ahh!! Thank you!!!! ❤️
Will correct now!